Modules/Registration/Common/test/itkMatchCardinalityImageToImageMetricTest.cxx

```diff
@@ -106,5 +106,13 @@ itkMatchCardinalityImageToImageMetricTest(int argc, char * argv[])
     }
   }
 
+
+  typename MetricType::ParametersType parameters = transform->GetParameters();
+  typename MetricType::DerivativeType derivative;
+  metric->GetDerivative(parameters, derivative);
+
+  typename MetricType::DerivativeType derivative1 = typename MetricType::DerivativeType{};
```

Comment:
In general, any variable declaration of the form `T var = T{}` could be simplified to just `T var{}`. So in this case:

    MetricType::DerivativeType derivative1{};

Reply:
Done in [c38c358](https://github.com/InsightSoftwareConsortium/ITK/pull/4060/commits/c38c3582da50e0a463da7f8174709cc1ad1c2180).